lib/resources/example_json/soarian_r4_examples_organization.rb

```diff
@@ -0,0 +1,340 @@
+          ],
+          "city": 'Louisville',
+          "state": 'KY',
+          "postalCode": '40201',
+          "country": 'United States',
+          "period": {
+            "start": '2021-01-08T09:58:45-05:00'
+          }
+        }
+      ],
+      "telecom": [
+        {
+          "system": 'phone',
+          "value": '+16102196300x9082',
+          "use": 'work',
+          "period": {
+            "start": '2020-12-04T09:36:34-05:00'
+          }
+        },
+        {
+          "system": 'phone',
+          "value": '6102196300',
+          "use": 'work',
+          "period": {
+            "start": '2014-04-10T15:20:38-04:00'
+          }
+        },
+        {
+          "system": 'phone',
+          "value": '6102196300',
+          "use": 'work',
+          "period": {
+            "start": '2014-04-10T15:21:06-04:00'
+          }
+        },
+        {
+          "system": 'phone',
+          "value": '610-219-6300',
+          "use": 'work',
+          "period": {
+            "start": '2014-04-10T15:21:50-04:00'
+          }
+        },
+        {
+          "system": 'phone',
+          "value": '610-219-6300',
+          "use": 'work',
+          "period": {
+            "start": '2014-04-10T15:22:16-04:00'
+          }
+        },
+        {
+          "system": 'phone',
+          "value": '212-555-9133',
+          "use": 'old',
+          "period": {
+            "start": '2021-01-08T09:58:45-05:00'
+          }
+        }
+      ],
+      "text": {
+        "status": 'generated',
+        # rubocop:disable Layout/LineLength
+        "div": "<div xmlns='http://www.w3.org/1999/xhtml'><b>Organization</b><br /><b>Name</b>:Health Enterprise;<br /><b>Status</b>:Active;<br /><b>Identifier</b>:<br /><b>System</b>:http://hl7.org/fhir/sid/us-npi <b>Value</b>:4510000001 ;<br /><b>Telecom</b>:<br /><b>System</b>:phone <b>Value</b>:+16102196300x9082 ;<br /><b>System</b>:phone <b>Value</b>:6102196300 ;<br /><b>System</b>:phone <b>Value</b>:6102196300 ;<br /><b>System</b>:phone <b>Value</b>:610-219-6300 ;<br /><b>System</b>:phone <b>Value</b>:610-219-6300 ;<br /><b>System</b>:phone <b>Value</b>:212-555-9133 ;<br /><b>Address</b>:<br />53 Valley Stream Parkway,Malvern,PA,19355,United States;<br />105 N 9th St,Philadelphia,PA,19100,United States;<br />208 Louisiana Ave,Philadelphia,PA,19101,United States;<br />51 Valley Stream Parkway,Philadelphia,PA,19101,United States;<br />PRM Address Street,Philadelphia,PA,19104,United States;<br />15 Wells Ln,Louisville,KY,40201,United States;<br /></div>"
+        # rubocop:enable Layout/LineLength
+      }
+    }.freeze
+
+    SOARIAN_R4_ORGANIZATION_SEARCH_BY_ID ||= {
+      "resourceType": 'Bundle',
+      "id": 'de340a64-b768-4d2a-ac8b-dc6496188539',
+      "type": 'searchset',
+      "timestamp": '2021-03-26T19:02:37-04:00',
+      "entry": [
+        {
+          "fullUrl": 'https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/Organization/HCU.1',
```

Comment:
Cross checking with Patient resource, we should ideally provide URLs which are public sandbox and are reachable. Patient resource has something like
https://fhir-myrecord-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/Patient/A879904FD2FE4B2D90C89FDA84E1285F'

Update in all other instances likewise